charts/kof-mothership/templates/sveltos/regional-multi-cluster-service.yaml

```
@@ -0,0 +1,62 @@
+apiVersion: k0rdent.mirantis.com/v1alpha1
+kind: MultiClusterService
+metadata:
+  name: kof-regional-cluster
+spec:
+  clusterSelector:
+    matchLabels:
+      k0rdent.mirantis.com/kof-cluster-role: regional
+    matchExpressions:
+      - key: "k0rdent.mirantis.com/istio-role"
+        operator: DoesNotExist
+
+  serviceSpec:
+    priority: 100
+    services:
+
+      - name: cert-manager
+        namespace: {{ .Release.Namespace }}
+        template: cert-manager-1-16-2
+        values: |
+          cert-manager:
+            crds:
+              enabled: true
+
+      - name: ingress-nginx
+        namespace: {{ .Release.Namespace }}
+        template: ingress-nginx-4-11-3
+
+      - name: kof-storage
+        namespace: {{ .Release.Namespace }}
+        template: kof-storage-{{ .Values.kcm.kof.charts.storage.version | replace "." "-" }}
```

We probably can change it to this, as all charts having the same version?
```suggestion
        template: kof-storage-{{ .Chart.Version | replace "." "-" }}
```